homeassistant/components/tuya/switch.py

```diff
@@ -24,6 +24,28 @@
 # default instruction set of each category end up being a Switch.
 # https://developer.tuya.com/en/docs/iot/standarddescription?id=K9i5ql6waswzq
 SWITCHES: dict[str, tuple[SwitchEntityDescription, ...]] = {
+    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH_DISTURB,
+            name="Do Not Disturb (DND)",
+            icon="mdi:minus-circle",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.VOICE_SWITCH,
+            name="Voice Switch",
+            icon="mdi:account-voice",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.RESET_MAP,
+            name="Map Resetting",
+            icon="mdi:map-marker-remove",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+        ),
```

Comment:
I don't think this is a real "switch", I think if it is toggled in the API, it toggles back immediately (it holds no state). This would be more fitting for a new entity we are adding, the `button`, however, that is not implemented in Tuya yet.

I suggest leaving it out for now. I'm adding a task to my list to add that platform to the Tuya integration.

```suggestion
```

Reply:
Agreed! I forgot about the button option, I also will remove this DP CODE from const.py.